### DISCUSSION TITLE
Html Export is missing the images

### DISCUSSION THREAD

**Discussion by hokutonanto** [TriliumNext/Trilium#1755] [Q&A]
When I try to export a note as HTML the path to the images looks something like this:
`api/images/Cu3c4F8Qgmmd/image.jpg`

and therefore all the images on the page are missing.
Is there a way to export the images with the correct path in the HTML?

**Comment by zadam** [comment]
Hi, I think the problem is that you're exporting a single note. An image is technically a separate note stored by default as a child note.

If you export the whole subtree ("this note and all its descendants") then the ZIP archive will contain the image and reference it correctly.
